app/src/main/java/com/example/movieapp/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.movies
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import by.kirich1409.viewbindingdelegate.viewBinding
+import com.example.movieapp.R
+import com.example.movieapp.databinding.FragmentMoviesDetailsBinding
+
+class FragmentMoviesDetails : Fragment(R.layout.fragment_movies_details) {
+    private val binding by viewBinding(FragmentMoviesDetailsBinding::bind)
+
+    override fun onCreateView(
```

Comment:
use onViewCreated instead of onCreateView

when u use onCreateView passing R.layout to constructor has no sense at all